content/applications/~dister-norsyr-torryn/canvas.md

```diff
@@ -0,0 +1,12 @@
++++
+title = "canvas"
+shortcode = "~dister-norsyr-torryn/canvas"
+license = "MIT"
+image = "https://yosoyubik.fra1.digitaloceanspaces.com/canvas/tile.svg"
+bgColor = "#3787B9"
+developer = "~dister-norsyr-torryn"
```

Comment:
The dev is `~norsyr-torryn` actually

Reply:
i've been conflating "developer" and "software provider" -- need to make a distinction between the two more clear and update some of the app pages and install guide

Reply:
The provider is in the shortcode; developer should be an entity or a patp right? Many ppl distribute via moons but it makes it hard to assess original author if both fields are the moon.

Reply:
Yes, the developer _should_ be an entity or patp.  However, in Landscape they use the terms "developer" to indicate an urbit address that could be a moon, star, anything, and "developer desk" to indicate the full shortcode.  So if we say "Tlon" is the developer then it's very confusing when "dozzod^dozzod" is also the developer.

Reply:
Chatting with James at Tlon now to see if how permanent that nomenclature is